library/kotlin/src/io/envoyproxy/envoymobile/Filter.kt

```diff
@@ -0,0 +1,7 @@
+package io.envoyproxy.envoymobile
+
+/*
+ * Interface representing a filter. See `RequestFilter` and `ResponseFilter` for more details.
+ */
+@Suppress("EmptyClassBlock")
```

Comment:
Let's add a TODO for avoid linting on this

Reply:
same as other comment